Package.swift

```diff
@@ -36,7 +41,11 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(url: "https://github.com/Esri/arcgis-runtime-ios", .upToNextMinor(from: "100.12.0"))
+        .package(
+            name: "arcgis-runtime-ios",
+            // path: sdk_install_package
+            url: "https://github.com/Esri/arcgis-runtime-ios", .upToNextMinor(from: "100.12.0")
+            )
```

Comment:
```suggestion
        .package(
            name: "arcgis-runtime-ios",
            // path: sdk_install_package
            url: "https://github.com/Esri/arcgis-runtime-ios", .upToNextMinor(from: "100.12.0")
        )
```
Indentation is off here. Use <kbd>control</kbd> <kbd>i</kbd> in Xcode to auto-reindent